SlaveWAMPServer.js

```diff
@@ -25,14 +25,14 @@ class SlaveWAMPServer extends WAMPServer {
 		this.endpoints.slaveRpc = {};
 		this.config = {};
 		this.internalRequestsTimeoutMs = internalRequestsTimeoutMs;
+		this.configuredCb = configuredCb;
```

Comment:
Just a note: If `configuredCb` is not used outside of the constructor, maybe we don't need to assign it to `this`?

Reply:
True, thanks!